package.json

```diff
@@ -58,12 +59,13 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-unicorn": "^11.0.2",
     "husky": "^9.0.0",
+    "jsdom": "^25.0.1",
```

Comment:
Unlike Jest, in Vitest there is no special "environment" package, but it expects JSDOM to be installed as a dependency. Before this change, it relied on the implicit version appeared in package-lock file, now it is explicit